ui/app/js/app.js

```diff
@@ -406,21 +406,21 @@ angular.module('am').config(
   function($routeProvider) {
     $routeProvider.
     when('/alerts', {
-      templateUrl: '/app/partials/alerts.html',
+      templateUrl: 'app/partials/alerts.html',
       controller: 'AlertsCtrl',
       reloadOnSearch: false
     }).
     when('/silences', {
-      templateUrl: '/app/partials/silences.html',
+      templateUrl: 'app/partials/silences.html',
       controller: 'SilencesCtrl',
       reloadOnSearch: false
     }).
     when('/status', {
-      templateUrl: '/app/partials/status.html',
+      templateUrl: 'app/partials/status.html',
       controller: 'StatusCtrl'
     }).
     otherwise({
       redirectTo: '/alerts'
```

Comment:
This redirect to `/alerts` won't work correctly when proxying to a different base URL.
(In our case `/alertmanager`...)